metrics/record_test.go

```diff
@@ -38,68 +38,87 @@ type cases struct {
-			metricsConfig: &metricsConfig{
-				isStackdriverBackend:        true,
-				stackdriverMetricTypePrefix: "knative.dev/internal/serving/activator",
-			},
-			measurement: measure.M(2),
-		}, {
-			name: "stackdriver backend with unsupported metric and allow custom metric",
-			metricsConfig: &metricsConfig{
-				isStackdriverBackend:        true,
-				stackdriverMetricTypePrefix: "knative.dev/unsupported",
-			},
-			measurement: measure.M(3),
-		}, {
-			name:        "empty metricsConfig",
-			measurement: measure.M(4),
+	// Increase the measurement value for each test case so that checking
+	// the last value ensures the measurement has been recorded.
+	shouldReportCases := []cases{{
+		name:          "none stackdriver backend",
+		metricsConfig: &metricsConfig{},
+		measurement:   measure.M(1),
+	}, {
+		name: "stackdriver backend with supported metric",
+		metricsConfig: &metricsConfig{
+			isStackdriverBackend:        true,
+			stackdriverMetricTypePrefix: "knative.dev/internal/serving/activator",
 		},
-	}
+		measurement: measure.M(2),
+	}, {
+		name: "stackdriver backend with unsupported metric and allow custom metric",
+		metricsConfig: &metricsConfig{
+			isStackdriverBackend:        true,
+			stackdriverMetricTypePrefix: "knative.dev/unsupported",
+		},
+		measurement: measure.M(3),
+	}, {
+		name:        "empty metricsConfig",
+		measurement: measure.M(4),
+	}}
 	testRecord(t, measure, shouldReportCases)
 }
 
 func TestRecordEventing(t *testing.T) {
 	measure := stats.Int64("event_count", "Number of event received", stats.UnitNone)
-	shouldReportCases := []cases{
-		// Increase the measurement value for each test case so that checking
-		// the last value ensures the measurement has been recorded.
-		{
-			name:          "none stackdriver backend",
-			metricsConfig: &metricsConfig{},
-			measurement:   measure.M(1),
-		}, {
-			name: "stackdriver backend with supported metric",
-			metricsConfig: &metricsConfig{
-				isStackdriverBackend:        true,
-				stackdriverMetricTypePrefix: "knative.dev/eventing/broker",
-			},
-			measurement: measure.M(5),
-		}, {
-			name: "stackdriver backend with unsupported metric and allow custom metric",
-			metricsConfig: &metricsConfig{
-				isStackdriverBackend:        true,
-				stackdriverMetricTypePrefix: "knative.dev/unsupported",
-			},
-			measurement: measure.M(3),
-		}, {
-			name:        "empty metricsConfig",
-			measurement: measure.M(4),
+	// Increase the measurement value for each test case so that checking
+	// the last value ensures the measurement has been recorded.
+	shouldReportCases := []cases{{
+		name:          "none stackdriver backend",
+		metricsConfig: &metricsConfig{},
+		measurement:   measure.M(1),
+	}, {
+		name: "stackdriver backend with supported metric",
+		metricsConfig: &metricsConfig{
+			isStackdriverBackend:        true,
+			stackdriverMetricTypePrefix: "knative.dev/eventing/broker",
 		},
-	}
+		measurement: measure.M(2),
```

Comment:
Why did this change from 5 to 2? (More generally, the reformatting on this file makes it hard to review.)

Reply:
comment above `// Increase the measurement value for each test` -- so I made comment reality :-)
I can split reformatting into a separate PR if you want, but this is superior formatting :-)